src/main/java/codeu/model/data/User.java

```diff
@@ -49,6 +51,16 @@ public String getName() {
     return name;
   }
 
+  /** Returns the User's about me statemt. */
+  public String getAboutMe() {
+    return aboutMe;
+  }
+
+  /** Changes the User's about me statemt. */
+  public void changeAboutMe(String newAboutMe) {
```

Comment:
nit: changeAboutMe -> setAboutMe ? Java typically uses get/set prefixes for member access like this. If this was more than simply setting the member variable directly then it would likely make more sense to deviate from this convention.